src/priorities/priorities.model.ts

```diff
@@ -0,0 +1,20 @@
+import { Column, DataType, HasMany, Model, Table } from 'sequelize-typescript';
+import { Demand } from '../demands/demands.model';
+
+@Table({ tableName: 'supplies', underscored: true })
+export class Priority extends Model {
+  @Column({ type: DataType.UUID, primaryKey: true, allowNull: false })
+  id!: string;
+
+  @Column({ allowNull: false, type: DataType.STRING })
```

Comment:
I guess you have to tell Sequelize that the column is not fully `snake_case` because you have the end of field with capital letters (`name_PL`). Does it work?

Reply:
I didn't test it here but had similar case in express app and Sequelize handled it properly but I changed the columns to small letters so it doesn't matter anymore :)